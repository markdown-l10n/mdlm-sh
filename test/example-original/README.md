## Example A

Here is a text of example.

### References

Some references here.
