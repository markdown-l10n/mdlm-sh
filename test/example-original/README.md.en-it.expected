**[English](README.md)** | [Italiano](README-it.md) <!-- l10n:select -->
## Example A

Here is a text of example.

### References

Some references here.
